static/app/utils/replays/getBreadcrumb.tsx

```diff
@@ -18,7 +20,12 @@ export function getPrevBreadcrumb({
     ) {
       return prev;
     }
-    if (!prev || crumbTimestampMS > +new Date(prev.timestamp || '')) {
+    if (
+      !prev ||
+      (allowEqual
+        ? crumbTimestampMS >= +new Date(prev.timestamp || '')
```

Comment:
This change is to avoid this behavior: 

Before:

![image](https://user-images.githubusercontent.com/39612839/185685050-0328df27-5c4d-4e91-8b60-e2cb4cad3c24.png)

After: 

![image](https://user-images.githubusercontent.com/39612839/185685116-fe794189-cea4-4d1e-853a-89b26589b940.png)